biz.ganttproject.core/src/biz/ganttproject/core/model/task/TaskDefaultColumn.java

```diff
@@ -76,6 +78,14 @@ private TaskDefaultColumn(ColumnList.Column delegate, Class<?> valueClass, Strin
     myNameKey = nameKey;
   }
 
+  public Comparator<?> getSortComparator() {
+    return mySortComparator;
+  }
+
+  public void setSortComparator(Comparator<?> mySortComparator) {
+    this.mySortComparator = mySortComparator;
```

Comment:
The purpose of prefixing with `my` is to indicate that it is a member variable without IDE assistance. Additional prefixing with `this.` is redundant: we already know it is a member variable.